flaskutils/models.py

```diff
@@ -1,8 +0,0 @@
-from pgsqlutils.orm import BaseModel
-
-
-class FlaskModel(BaseModel):
```

Comment:
you can use the last release wich corresponds to the last version before this merge